src/domain.py

```diff
@@ -350,10 +359,31 @@ def domain_add(
             pass
         raise e
 
+    failed_letsencrypt_cert_install = False
+    if install_letsencrypt_cert:
+        parent_domain = _get_parent_domain_of(domain)
+        can_install_letsencrypt = (
+            parent_domain
+            and certificate_status([parent_domain], full=True)["certificates"][
+                parent_domain
+            ]["has_wildcards"]
+        )
+
+        if can_install_letsencrypt:
+            try:
+                _certificate_install_letsencrypt([domain], force=True, no_checks=True)
+            except:
+                failed_letsencrypt_cert_install = True
+        else:
+            failed_letsencrypt_cert_install = True
```

Comment:
```suggestion
            logger.warning("Skipping Let's Encrypt certificate attempt because there's no wildcard configured on the parent domain's DNS records.")
            failed_letsencrypt_cert_install = True
```